packages/clay-shared/src/__tests__/FocusScope.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {cleanup, render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import '@testing-library/jest-dom/extend-expect';
+import {ClayPortal, FocusScope} from '..';
+
+describe('FocusScope', () => {
+	afterEach(() => {
+		jest.clearAllTimers();
+
+		cleanup();
+	});
+
+	it('manages focus order when using portals', () => {
```

Comment:
This looks good, I think it's a good start for us to be able to add more cases here.